sdk/tables/azure-data-tables/samples/async_samples/sample_insert_delete_entities_async.py

```diff
@@ -37,7 +39,17 @@ def __init__(self):
         )
         self.table_name = "InsertDeleteAsync"
 
-        self.entity = {"PartitionKey": "color", "RowKey": "brand", "text": "Marker", "color": "Purple", "price": "5"}
+        self.entity = {
+            "PartitionKey": "color",
+            "RowKey": "brand",
+            "text": "Marker",
+            "color": "Purple",
+            "price": 4.99,
+            "last_updated": datetime.today(),
+            "product_id": uuid4(),
+            "inventory_count": 42,
+            "some_binary_value": b"somebinaryvalue"
```

Comment:
Haha - It looks like the marker merchandise theme petered out a bit :P 
How about:
```python
"barcode": b"somebinaryvalue"
```

Reply:
Haha thank you I could not think of a way to use binary with the marker theme.